src/SourceBrowser/src/SourceIndexServer/wwwroot/overview.html

```diff
@@ -148,5 +148,19 @@ <h3>Features</h3>
             </td>
         </tr>
     </table>
+<footer class="text-muted fade-in">
+    <div class="container mt-5">
+        <p class="float-right">
+            <a href="#">Back to top</a>
+        </p>
+        <p>
+            <span>
+                <a href="mailto:dnceng@microsoft.com">Contact Us</a> |

```

Comment:
Should this be the new owners already?

Reply:
It hasn't technically been transferred yet. I have a hand-off note to update this address when that happens.